### DISCUSSION TITLE
Update of CRDs during operator upgrade.

### DISCUSSION THREAD

**Discussion by zumbuschms** [OT-CONTAINER-KIT/redis-operator#1010] [Q&A]
When I upgrade the redis-operator to a new version, the CRDs are not automatically updated. Is there a command line parameter for helm upgrade that updates the CRDs during the upgrade?

The only solution I was able to find so far is a complete uninstall of the operator and manual deletion of the CRDs and then reinstall the oeprator.

**Comment by drivebyer** [comment]
CRD upgrades are not supported in Helm. This is an important operation that needs to be handled manually. Perhaps we can find some best practices from other operators and learn from them. @zumbuschms

**Comment by shubham-cmyk** [comment]
@zumbuschms Yes you are right. 
You have to either take the CRD and apply the field manually use `kubectl` with `--server-side`
